docs/test/live-unit-testing-faq.md

```diff
@@ -110,6 +120,45 @@ This ensures that `<OutputPath>` lies within the `<BaseOutputPath>` folder.
 
 Do not override `<OutDir>` directly in your build process; override `<OutputPath>` instead to drop build artifacts to a specific location.
 
+### Overriding your properties based on the `<LiveUnitTestingBuildRootPath>` property.
+
+> Note: In this approach you need to be careful about files added under the artifacts folder that are not generated during build. The example below shows what to do with placing the packages folder under artifacts. Since the contents of this folder are not generated during the build the MSBuild property **should not be changed**.
```

Comment:
```suggestion
> [!NOTE]
> In this approach, you need to be careful about files added under the artifacts folder that are not generated during build. The example below shows what to do when placing the packages folder under artifacts. Because the contents of this folder are not generated during the build, the MSBuild property **should not be changed**.
```